autoload/gina/complete/common.vim

```diff
@@ -41,6 +41,42 @@ function! gina#complete#common#command(arglead, cmdline, cursorpos) abort
   return gina#util#filter(a:arglead, command_names, '^_')
 endfunction
 
+function! gina#complete#common#raw_command(arglead, cmdline, cursorpos) abort
+  return gina#util#filter(a:arglead, [
+        \ 'add',
+        \ 'bisect',
+        \ 'branch',
+        \ 'checkout',
+        \ 'clone',
+        \ 'commit',
+        \ 'diff',
+        \ 'fetch',
+        \ 'grep',
+        \ 'init',
+        \ 'log',
+        \ 'merge',
+        \ 'mv',
+        \ 'pull',
+        \ 'push',
+        \ 'rebase',
+        \ 'reset',
+        \ 'restore',
+        \ 'rm',
+        \ 'show',
+        \ 'status',
+        \ 'switch',
+        \ 'tag',
+        \])
```

Comment:
Taken from the `__git_zsh_cmd_common ()` function in https://opensource.apple.com/source/Git/Git-58/src/git/contrib/completion/git-completion.zsh.auto.html.